src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchSequenceChecksums.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import { gql } from '@apollo/client';
+import { Sequence } from 'ensemblRoot/src/shared/types/thoas/sequence';
```

Comment:
Separate this import from vendor imports i.e. `gql`?

Reply:
Done. @azangru and @imransl can you please review. Thanks